src/components/Comments/Comment.js

```diff
@@ -190,7 +271,7 @@ const Comment = ({
           </ButtonHightlight>
         </TouchableOpacity>
 
-        <Text style={styles.vote(voteStyle())}>{totalVote}</Text>
+        {totalVote != 0 && <Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition check for `totalVote != 0` is a good addition to prevent unnecessary rendering of the vote count when it's zero.



<!-- This is an auto-generated comment by OSS CodeRabbit -->